### ISSUE TITLE
Bad indention when examples from external file used

### ISSUE THREAD

**Issue by LukasSefcik** [dotnet/docfx#2830]
**DocFX Version Used**: 2.36.1.0

**Template used**: `default`

**Steps to Reproduce**:

1. I added examples from external file.
`<code source="..\..\Examples\Kros.KORM.Examples\IQueryExample.cs" title="Query data by query builder" region="Select" lang="c#"  />`
2. Build documentation.

Example file looks like this:

```csharp
namespace KORM.Test.Performance.Doc
{
    internal class IQueryExample
    {
        public void SelectExample()
        {
            using (var database = new Database(new SqlConnection()))
            {
                #region Select
                var people = database.Query<Person>()
                    .Select("p.Id", "FirstName", "LastName", "PostCode")
                    .From("Person JOIN Address ON (Person.AddressId = Address.Id)")
                    .Where("Age > @1", 18);

                foreach (var person in people)
                {
                    Console.WriteLine(person.FirstName);
                }
                #endregion
            }
        }
    }
}
```

**Expected Behavior**:
Indention retrieved from the source file start from 0 indention.

**Actual Behavior**:
The indention is retrieved from the source file:
![docfx_example](https://user-images.githubusercontent.com/24602626/40901276-cf012c00-67cf-11e8-8dea-ac9f2853f22a.png)

**Comment by vwxyzh** [comment]
@LukasSefcik is there mixed tabs and spaces in your source code.

**Comment by LukasSefcik** [comment]
@vwxyzh No, I only use spaces.

**Comment by herohua** [comment]
@LukasSefcik 
This bug is fixed in v2.36.2. Can you kindly verify? Thanks!

**Comment by LukasSefcik** [comment]
@herohua 
Still bad indention. I have installed version 2.36.2 via chocolatey.
